saltstack/final/salt/cleanup/tmp/generate-package-versions.sh

```diff
@@ -2,6 +2,18 @@
 
 set -x
 
+set_version_for_rpm_pkg() {
+	package_name=$1
+	package_installed=$(rpm -q "$package_name" 2>&1 >/dev/null; echo $?)
+	if [[ "$package_installed" == "0" ]]; then
+		rpm_version=$(rpm -q --queryformat '%-{VERSION}' "$package_name")
+		echo "{\"$package_name\": \"$rpm_version\"}" > /tmp/add_pkg_version.json.tmp
+		jq -s '.[0] * .[1]' /tmp/add_pkg_version.json.tmp /tmp/package-versions.json > /tmp/package-versions.json.tmp
+		rm -f /tmp/add_pkg_version.json.tmp
+		mv /tmp/package-versions.json.tmp /tmp/package-versions.json
```

Comment:
question: is `/tmp/package-versions.json` purposefully overwritten in line 17 and 18?